src/Appwrite/Platform/Workers/Databases.php

```diff
@@ -480,27 +455,179 @@ private function deleteIndex(Document $database, Document $collection, Document
                 $index->setAttribute('status', 'stuck')
             );
         } finally {
-            $target = Realtime::fromPayload(
-            // Pass first, most verbose event pattern
-                event: $events[0],
-                payload: $index,
-                project: $project
-            );
-
-            Realtime::send(
-                projectId: 'console',
-                payload: $index->getArrayCopy(),
-                events: $events,
-                channels: $target['channels'],
-                roles: $target['roles'],
-                options: [
-                    'projectId' => $projectId,
-                    'databaseId' => $database->getId(),
-                    'collectionId' => $collection->getId()
-                ]
-            );
+            $this->trigger($database, $collection, $index, $project, $projectId, $events);
         }
 
         $dbForProject->deleteCachedDocument('database_' . $database->getInternalId(), $collection->getId());
     }
+
+    /**
+     * @param Document $database
+     * @param Document $project
+     * @param $dbForProject
+     * @return void
+     * @throws Exception
+     */
+    protected function deleteDatabase(Document $database, Document $project, $dbForProject): void
```

Comment:
Shouldn't we now remove this code from the deletes worker ?

Reply:
I think we can.